index.html

```diff
@@ -0,0 +1,386 @@
+                  src="images/Counter.png" alt="counter"></span>
+              <span class="inrow"> 2015</span>
+            </p><br>
+            <p class="innerpara">A daily selection of privately personalized reads; no accounts or sign-ups
+              required.</p><br>
+            <ul>
+              <li class="back">html</li>
+              <li class="back">css</li>
+              <li class="back">javascript</li>
+            </ul><br>
+            <button type="button" class="button button4">See Project</button>
+          </div>
+        </div>
+      </div>
+    </section>
+  </div>
+
+  <div class="container">
+    <section class="back2">
+      <div class="item1">
+
+        <p class="bold2">About<span><br>Myself</span></p>
+        <p class="customise customise1">Hello I’m a software developer! I can help you build a product , feature or website Look
+          through some of my work and experience! If you like what you see and have a project you need coded, don’t
+          hestiate
+          to contact me.</p>
+        <h3 class="letsconnect">LET'S CONNECT</h3>
+        <div class="social-media">
+          <ul class="onerow1">
+            <li><a href="https://twitter.com/"><i class="fa-brands fa-twitter"></i></a></li>
+            <li><a href="https://www.linkedin.com/"><i class="fa-brands fa-linkedin-in"></i></a></li>
+            <li><a href="https://medium.com/"><i class="fa-brands fa-medium"></i></a></li>
+            <li><a href="https://github.com/"><i class="fa-brands fa-github"></i></a></li>
+            <li><a href="https://angel.co/"><i class="fa-brands fa-angellist"></i></a></li>
+          </ul>
+        </div>
+        <br>
+        <button type="button" class="button button4">Get my resume</button>
+      </div>
+
+      <div class="item2">
+        <div class="languages2">
+          <h4 class="languages1">Languages</h4>
+          <p class="arrow"><img src="images/Vector1.png" alt="Down arrow">
+          </p>
+        </div>
+        <div class="lang2">
+          <ul class="nodots">
+            <li>
+              <div class="langcolor"><img src="images/Frame9.png" alt="js image">
+              </div>
+            </li>
+            <li>
+              <div class="langcolor"><img src="images/Frame10.png" alt="html5 image">
+              </div>
+            </li>
+            <li>
+              <div class="langcolor"><img src="images/Frame11.png" alt="css3 image">
+            </li>
+          </ul>
+        </div>
+        <div class="lang1">
+          <ul class="nodots1">
+            <li>
+              <p class="js">Frameworks<span class="arrow"><img src="images/right.jpg" alt="right arrow"> </span></p>
+            </li>
+            <li>
+              <hr>
+            </li>
+            <li>
+              <p class="js">Skills<span class="arrow1"><img src="images/right.jpg" alt="right arrow"> </span></p>
```

Comment:
- [ ] Please check your images because they are not working
your design
![image](https://user-images.githubusercontent.com/53402600/173161573-2b496cac-ffbb-4b8f-9ed4-6a843e868420.png)